src/theme/css/wiki-themes/breadcrumbs.css

```diff
@@ -1,25 +1,33 @@
 .breadcrumb {
-    display: flex;
-    flex-wrap: wrap;
-    margin-top: 5px;
-    color: var(--links);
+    display: -webkit-box;
```

Comment:
Why are you using `-webkit-box`? Is it supported by all major browsers? And more crucially for this wiki, mobile browsers?

Also, isn't it just old syntax for `flexbox`?

Reply:
Safari, Edge, Yandex, Google and Chrome work without errors.
Why? Because then design would look really awful...
![image](https://github.com/user-attachments/assets/d5cb7501-4483-4ff1-afa4-18e301c0e371)

Reply:
oh yeah, this is old version of `flexbox`

Reply:
but with `flexbox`:
![image](https://github.com/user-attachments/assets/26d42cc7-0696-43bc-9c03-bcd869eb56e0)

Reply:
Try `box`? Honestly, no idea. I don't know much about CSS, but try avoiding any non-standard properties.

If any other value doesn't work like `-webkit-box` then just leave it as is, just let me know. I'm sure there should be one without the `-webkit-` prefix

Reply:
From my quick testing now, `block ruby` looks like works like `-webkit-box` but it is also on the list in dev tools so I guess it can stay. 🤷